modules/loki-rbac-permissions.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * logging/logging_alerts/custom-logging-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="loki-rbac-permissions_{context}"]
+= Authorizing Loki rules RBAC permissions
+
+In logging 5.8 and later, the Cluster Logging Operator provides `alertingrule-editor-role` and `recordingrule-editor-role` cluster roles, which enable users to modify alerting and recording rules for the LokiStack.
+
+You can authorize RBAC permissions for modifying rules by binding the required cluster roles to a username.
+
+.Prerequisites
+
+* The Cluster Logging Operator is installed in the `openshift-logging` namespace.
+* You have administrator permissions.
+
+.Procedure
+
+* Bind the appropriate cluster roles to the user:
++
+.Example binding command
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user <cluster_role_name> -n <namespace> <username>
```

Comment:
Not giving more permissions than what the user needs. Similar to how the monitoring team has in https://65341--docspreview.netlify.app/openshift-enterprise/latest/monitoring/enabling-monitoring-for-user-defined-projects#granting-user-permissions-using-the-cli_enabling-monitoring-for-user-defined-projects
```suggestion
$ oc adm policy add-role-to-user <cluster_role_name> -n <namespace> <username>
```